Revert "upgrade metallb to v0.13.10 (#214)"

bschimke95 opened the pull request:
Revert "upgrade metallb to v0.13.10 (#214)"
    
This reverts commit a33482b8daef3b81e311decb567af2162b76dbff.
    
The Metallb upgrade to v0.13.10 seems to cause issues. Revert this
upgrade until we investigated this issue.
    
Closes MK-1382

neoaggelos commented:
For reference, two of the issues we are seeing are:

- The `validatingwebhookconfiguration` was renamed, therefore causing issues for some users.
- LoadBalancer IPs are sometimes not properly advertised on the network, making it impossible to use the services.

neoaggelos commented:
@bschimke95 could you please also include the following logic in the `disable` script of the addon, so that we can deal with the webhook issues:

```bash
if kubectl get validatingwebhookconfiguration -o yaml metallb-webhook-configuration 2>/dev/null | grep -q metallb.io; then
  kubectl delete validatingwebhookconfiguration metallb-webhook-configuration
fi

if kubectl get validatingwebhookconfiguration -o yaml validating-webhook-configuration 2>/dev/null | grep -q metallb.io; then
  kubectl delete validatingwebhookconfiguration validating-webhook-configuration
fi
```

neoaggelos reviewed (commented):
Please see comment about extending the `disable` hook to mitigate some of the issues that people might have run into. We also have to backport this to 1.28 after merging.

bschimke95 commented:
* Added a new commit with the suggested updates for the `disable` script (new commit to preserve pure 'revert' commit)
* Changed base branch to main (will create a backport once this is merged).
